.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -22,7 +22,7 @@ the PR to a particular issue, see
     - [ ] Commits have useful messages
     - [ ] The documentation has been properly updated
     - [ ] New tests are added if needed and existing tests are updated
-    - [ ] Any changes affecting Consensus packages must have an entry in the appropriate `changelog.d` directory created using [`scriv`](https://github.com/input-output-hk/scriv). If in doubt, see the [Consensus release process](../ouroboros-consensus/docs/ReleaseProcess.md).
+    - [ ] Any changes affecting Consensus packages must have an entry in the appropriate `changelog.md` directory created using [`scriv`](https://github.com/input-output-hk/scriv). If in doubt, see the [Consensus release process](../ouroboros-consensus/docs/ReleaseProcess.md).
```

Comment:
I think this should remain as is, namely `changelog.d`.

Reply:
My bad, should have looked into the actual files. Never seen the `scriv`/`.d` before.

Will revert